ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -109,18 +109,21 @@ private enum UnityLayers {
   private bool navball_changed_ = true;
   private FlightGlobals.SpeedDisplayModes? previous_display_mode_;
 
-  private UnityEngine.Color history_colour = XKCDColors.Lime;
-  private GLLines.Style history_style = GLLines.Style.Faded;
-  private UnityEngine.Color prediction_colour = XKCDColors.Fuchsia;
-  private GLLines.Style prediction_style = GLLines.Style.Solid;
-  private UnityEngine.Color flight_plan_colour = XKCDColors.PeriwinkleBlue;
-  private GLLines.Style flight_plan_style = GLLines.Style.Dashed;
-  private UnityEngine.Color burn_colour = XKCDColors.Pink;
-  private GLLines.Style burn_style = GLLines.Style.Solid;
-  private UnityEngine.Color target_history_colour = XKCDColors.Goldenrod;
-  private GLLines.Style target_history_style = GLLines.Style.Faded;
-  private UnityEngine.Color target_prediction_colour = XKCDColors.LightMauve;
-  private GLLines.Style target_prediction_style = GLLines.Style.Solid;
+  // TODO(egg): move all of that to the plotter once the RP2 lines are gone.
```

Comment:
When is that?

Reply:
Now. But let’s deal with that in another PR.